content/en/security_monitoring/detection_rules.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Security Detection Rules
+kind: documentation
+further_reading:
+- link: "/security_monitoring/explorer"
+  tag: "Documentation"
+  text: "Search through all of your security signals and perform Security Analytics"
+---
+
+## Overview
+
+Use Detection Rules to define conditional logic that is applied to your entire event stream. When at least one case defined in a Detection Rule is matched over a given period of time, Datadog generates a Security Signal.
+
+## Create Detection Rules
+
+To create a Detection Rule in Datadog, use the main navigation: **Security** → **Detection Rules** --> **New Rule**.
+
+### Define the search query
+
+{{< img src="security_monitoring/detection_rules/define_search_query.png" alt="Define the search query" responsive="true">}}
+
+Construct a search query using the same logic as a [log explorer search][1]. Note that each query has a label, which is a lowercase ASCII letter and is not user controllable.
+
+Optionally, define an alert grouping. The defined grouping is used to reduce the number of signals generated. The group-by is also used to join the queries together.
+
+Add additional queries with the **Add Query** button. 
+
+
+### Define the rule cases
+
+{{< img src="security_monitoring/detection_rules/set_rule_cases.png" alt="Set rule cases" responsive="true">}}
+
+#### Rule case trigger and name
+
+Rule cases, such as `a > 3`, are evaluated as case statements. Thus, the first case to match generates the signal. Click and drag your rule cases to manipulate their ordering.
+
+A rule case contains logical operations (`>, <, >=, <=, &&, ||, !`) to determine if a signal should be generated based on the event counts in the previously defined queries. The ASCII lowercase [query labels](#define-the-search-query) are referenced in this section. 
```

Comment:
`>, <, >=, <=, &&, ||, !` should be `>,  >=, &&, ||`

Also, there should be there should be some additional text related to rule case that explains the following:

> The query label must be before the operator. Example: `a > 3`. The following is NOT allowed: `3 > a`